test/utils.ts

```diff
@@ -46,15 +52,20 @@ function compileLib(name, options = {}) {
  * @param {number} expected - Expected value.
  * @param {string} [message] - Debug message.
  */
-function assertNumberEqual(actual, expected, message = undefined) {
-  if (Number.isNaN(expected)) {
-    expect(actual, message).NaN;
-  } else {
-    expect(actual, message).eq(expected);
+function assertNumberEqual(actual: number, expected: number, message?: string) {
+  try {
+    if (Number.isNaN(expected)) {
+      expect(actual).toBeNaN();
+    } else {
+      expect(actual === expected).toBeTruthy();
+    }
+  } catch (e) {
+    if (message) throw new Error(message);
+    else throw e;
   }
 }
 
-function compareResults(x, y) {
+function compareResults(x: number, y: number) {
```

Comment:
`x: number | number[], y: number | number[]`

Sorry for the lack of comments :|

Reply:
Fixed. Thanks 👍